explore/src/main/scala/explore/targeteditor/AladinCell.scala

```diff
@@ -89,73 +89,67 @@ object AladinCell extends ModelOptics {
       // flag to trigger centering. This is a bit brute force but
       // avoids us needing a ref to a Fn component
       .useStateView(false)
-      // to get faster reusability use a serial state, rather than check every candidate
-      .useSerialState(List.empty[GuideStarCandidate])
```

Comment:
It seems we don't need to do this anymore since we are not using `Reusability` and that we can use the `useStreamResource` result directly.

Reply:
Don't we need it for the `useMemoBy` below?

Reply:
Yes, you are right, I missed that.

However, I was just about to roll this back but I'm doubting if it is really necessary: the list of candidates is only updated once per query to the worker, right?

Reply:
Right it should be only once. the list can be fairly large, omega cen has 17k target and doing a Reusability comparison would need to traverse the 17k items right?
That's why I thought about serial state but maybe we don't need it

Reply:
Wait, if one of the other `useMemo` dependencies changes then, yes, the canonic list reusability will be used for candidates.

I'll roll back.